scripts/commands/checkLinks.ts

```diff
@@ -203,19 +218,44 @@ async function determineDevFileBatches(): Promise<FileBatch[]> {
 
 async function determineHistoricalFileBatches(
   projectName: string,
-  toLoad: string[] = [],
+  extraFilesToLoad: string[],
+  checkHistoricalApiDocs: boolean = true,
+  checkSeparateReleaseNotes: boolean = false,
 ): Promise<FileBatch[]> {
+  if (!checkHistoricalApiDocs && !checkSeparateReleaseNotes) {
+    return [];
+  }
+
   const historicalFolders = (
     await readdir(`docs/api/${projectName}`, { withFileTypes: true })
   ).filter((file) => file.isDirectory() && file.name.match(/[0-9].*/));
 
   const result = [];
   for (const folder of historicalFolders) {
-    const fileBatch = await FileBatch.fromGlobs(
-      [
+    const toCheck: string[] = [];
+    const toLoad = [...extraFilesToLoad];
+
+    if (checkHistoricalApiDocs) {
```

Comment:
If `checkSeparateReleaseNotes` is set but not `checkHistoricalApiDocs`, then you need to add ``docs/api/${projectName}/${folder.name}/*`,` in `toLoad`.